File: Basics_with_JavaScript/78 Conditional Statements Advanced/More Exersises/08. Point on Rectangle Border.js
function vladi(input) {
  let x1 = Number(input[0]);
  let y1 = Number(input[1]);
  let x2 = Number(input[2]);
  let y2 = Number(input[3]);
  let x = Number(input[4]);
  let y = Number(input[5]);
  isBorder = false;

  if (x === x1 || x === x2) {
    if (y >= y1 && y <= y2) {
      isBorder = true;
    } else {
      isBorder = false;
    }
  } else if (y === y1 || y === y2) {
    if (x >= x1 && x <= x2) {
      isBorder = true;
    } else {
      isBorder = false;
    }
  } else {
    isBorder = false;
  }

  if (isBorder) {
    console.log("Border");
  } else {
    console.log("Inside / Outside");
  }
}

vladi(["2", "-3", "12", "3", "8", "-1"]);
vladi(["2", "-3", "12", "3", "12", "-1"]);

File: Basics_with_Python/PythonBasicCourse/3_Conditional_Statements_Advanced/8_Conditional_Statements_Advanced_Exercise_Lab/more_exercises/08._Point_on_Rectangle_Border.py
x1 = float(input())
y1 = float(input())
x2 = float(input())
y2 = float(input())
x = float(input())
y = float(input())
border = False

if (x == x1 and y1 <= y <= y2) or (x == x2 and y1 <= y <= y2) or (y == y1 and x1 <= x <= x2) or (y == y2 and x1 <= x <= x2):
    print("Border")
else:
    print("Inside / Outside")
